README.md

```diff
@@ -53,6 +53,7 @@ opx-build/scripts/opx_run opx_build opx-logging
 # Build against the 2.2.1 release
 OPX_RELEASE=2.2.1 opx-build/scripts/opx_run opx_build all
 ```
+**Note:** To run a container based on Stretch, make sure the already existing Docker image named "opxhub/build" is Stretch based if not just remove it.
```

Comment:
Change this to:

**Note**: Ensure the Docker image is up to date by running `docker pull opxhub/build`.

Reply:
Thanks